### ISSUE TITLE
torch.Softmax only accepts 2D tensors

### ISSUE THREAD

**Issue by keerthanpg** [pytorch/pytorch#3319]
The documentation says it can accept a tensor of any dimension and input dimensions and output dimensions match. However, when you give it a 1D tensor, it says it expects 2D tensor. I am using pytorch with python3 and ubuntu 16.04.

**Comment by keerthanpg** [comment]
Correction. I mean torch.nn.Softmax()

**Comment by fmassa** [comment]
Thais functionality was very recently added to pytorch (last week), and is not yet present in any of the pre-compiled binaries (like conda). What's your pytorch version? `torch.__version__`?

**Comment by keerthanpg** [comment]
torch-0.2.0 Not sure.
My installation command looked like pip3 install http://download.pytorch.org/whl/cu80/torch-0.2.0.post3-cp35-cp35m-manylinux1_x86_64.whl

**Comment by SsnL** [comment]
You are looking at the wrong documentation. The correct one for 0.2.0 is at http://pytorch.org/docs/0.2.0/nn.html#torch.nn.Softmax, which specifically says 2D.
